tests/Features/Expect/toHaveProperties.php

```diff
@@ -4,30 +4,49 @@
 
 test('pass', function () {
     $object = new stdClass();
-    $object->name = 'Jhon';
+    $object->name = 'John';
     $object->age = 21;
 
-    expect($object)->toHaveProperties(['name', 'age']);
+    expect($object)
+        ->toHaveProperties(['name', 'age'])
+        ->toHaveProperties([
+            'name' => 'John',
+            'age' => 21,
+        ]);
 });
 
 test('failures', function () {
     $object = new stdClass();
-    $object->name = 'Jhon';
-
-    expect($object)->toHaveProperties(['name', 'age']);
+    $object->name = 'John';
+
+    expect($object)
+        ->toHaveProperties(['name', 'age'])
```

Comment:
@lucasgiovanny maybe this group of failing tests could be split into two separate tests ? The second expectation could not fail and the exception will be thrown anyway because of the first one